src/server/game/Spells/SpellInfo.cpp

```diff
@@ -920,7 +920,7 @@ SpellEffectInfo::StaticData SpellEffectInfo::_data[TOTAL_SPELL_EFFECTS] =
     {EFFECT_IMPLICIT_TARGET_NONE,     TARGET_OBJECT_TYPE_NONE}, // 195 SPELL_EFFECT_195
     {EFFECT_IMPLICIT_TARGET_NONE,     TARGET_OBJECT_TYPE_NONE}, // 196 SPELL_EFFECT_196
     {EFFECT_IMPLICIT_TARGET_NONE,     TARGET_OBJECT_TYPE_NONE}, // 197 SPELL_EFFECT_197
-    {EFFECT_IMPLICIT_TARGET_NONE,     TARGET_OBJECT_TYPE_NONE}, // 198 SPELL_EFFECT_PLAY_SCENE
+    {EFFECT_IMPLICIT_TARGET_CASTER,   TARGET_OBJECT_TYPE_UNIT_AND_DEST}, // 198 SPELL_EFFECT_PLAY_SCENE
```

Comment:
So 
{EFFECT_IMPLICIT_TARGET_NONE,     TARGET_OBJECT_TYPE_UNIT_AND_DEST}
or
{EFFECT_IMPLICIT_TARGET_NONE,     TARGET_OBJECT_TYPE_DEST}

I havent really looked much into it and just made sure it still worked in game :/